static/bidder-info/boldwin.yaml

```diff
@@ -13,3 +13,7 @@ capabilities:
     - banner
     - video
     - native
+userSync:
+  redirect:
+    url: "https://sync.videowalldirect.com/sync/prebid?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&ccpa={{.USPrivacy}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
@Wls-demo 

tested following cases but non of them worked

```
https://sync.videowalldirect.com/sync/prebid?gdpr=0&gdpr_consent=0&ccpa=1&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%boldwin%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

https://sync.videowalldirect.com/sync/prebid?gdpr=0&gdpr_consent=1&ccpa=0&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%boldwin%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

https://sync.videowalldirect.com/sync/prebid?gdpr=0&gdpr_consent=1&ccpa=1&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%boldwin%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D


https://sync.videowalldirect.com/sync/prebid?gdpr=1&gdpr_consent=0&ccpa=0&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%boldwin%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

https://sync.videowalldirect.com/sync/prebid?gdpr=1&gdpr_consent=0&ccpa=1&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%boldwin%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

https://sync.videowalldirect.com/sync/prebid?gdpr=1&gdpr_consent=1&ccpa=0&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%boldwin%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D


https://sync.videowalldirect.com/sync/prebid?gdpr=1&gdpr_consent=1&ccpa=1&redir=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%boldwin%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%5BUID%5D

```

Reply:
sorry, was error with url path. changed

Reply:
user sync url is working after changes from https://github.com/prebid/prebid-server/pull/3316/commits/9daee8237a2f1574ff699189906277c03f936ae1